ouroboros-consensus/changelog.d/20250219_054827_nick.frisby_simplify_inject.md

```diff
@@ -0,0 +1,24 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
+### Patch
+
+- A bullet item for the Patch category.
+
+-->
+<!--
+### Non-Breaking
+
+- A bullet item for the Non-Breaking category.
+
+-->
+
+### Breaking
+
+- Make the type of `Nary.inject` more precise.
+  The old type involves oracular data, so any required changes downstream are almost certainly limited to testing code.
+  (I'm therefore tempted to list this as Patch instead --- violating the [PVP](https://pvp.haskell.org/) --- but the risk-reward seems prohibitive.)
```

Comment:
:+1: (in particular, almost all of our releases are major anyways, so it is unlikely that this will be responsible for a "spurious" major version bump)